packages/react/src/analytics/integrations/Guardian/providers/__tests__/Riskified.test.ts

```diff
@@ -0,0 +1,125 @@
+import { EnvironmentTypes, integrations } from '@farfetch/blackout-analytics';
+import { Riskified } from '..';
+
+describe('Riskified', () => {
+  const defaultKey = 'farfetch.com';
+  const baseUrl = (key?: string) =>
+    `https://beacon.riskified.com?shop=${key || defaultKey}&sid=`;
+  const localId = '3c11a6f9-6d5e-45c4-b1b2-d26736964605';
+
+  const loadData = {
```

Comment:
Instead of creating a new loadData mock, there is a fixture in analytics for this exact reason in `loadIntegrationData.fixtures.ts` that you can reuse here.

Reply:
fixed